ebill_paynet/messages/invoice-2013A.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<XML-FSCM-INVOICE-2013A>
+  <INTERCHANGE>
+    <IC-SENDER>
+      <Pid>{{ client_pid }}</Pid>
+    </IC-SENDER>
+    <IC-RECEIVER>
+      <Pid>41010106799303734</Pid>
+    </IC-RECEIVER>
+    <IC-Ref>{{ ic_ref }}</IC-Ref>
+  </INTERCHANGE>
+  <INVOICE Type="{{ document_type }}">
+    <HEADER>
+      <FUNCTION-FLAGS>
+        <Confirmation-Flag />
+      </FUNCTION-FLAGS>
+      <MESSAGE-REFERENCE>
+        <REFERENCE-DATE>
+          <Reference-No>{{ invoice.name }}</Reference-No>
+          <Date>{{ format_date() }}</Date>
+        </REFERENCE-DATE>
+      </MESSAGE-REFERENCE>
+      <PRINT-DATE>
+        <Date>{{ format_date(invoice.date_invoice) }}</Date>
+      </PRINT-DATE>
+      <REFERENCE>
+        <INVOICE-REFERENCE>
+          <REFERENCE-DATE>
+            <Reference-No>{{ invoice.name }}</Reference-No>
+            <Date>{{ format_date(invoice.date_invoice) }}</Date>
+          </REFERENCE-DATE>
+        </INVOICE-REFERENCE>
+      </REFERENCE>
+      <BILLER>
+        {#- Doc says vat number or business identitfiaction number #}
+        {%- if invoice.company_id.vat %}
+        <Tax-No>{{ invoice.company_id.vat }}</Tax-No>
+        {%- endif %}
+        {%- if invoice_esr %}
+            <Doc-Reference Type="ESR-NEU">{{ invoice_esr }}</Doc-Reference>
```

Comment:
```suggestion
            <Doc-Reference Type="ESR-NEU">{{ invoice_isr }}</Doc-Reference>
```

Reply:
This file is related to Paynet B2C service and is not implemented yet but kept for further reference